docs/cluster-configuration.md

```diff
@@ -177,6 +177,12 @@ For additional information, see here: [KubeSecondaryDNS](https://github.com/kube
 
 **Default**: `false`
 
+### deployKubevirtIpamController Feature Gate
+Set the `deployKubevirtIpamController` feature gate to true to allow deploying Kubevirt IPAM controller by CNAO.
+For additional information, see here: [KubevirtIpamController](https://github.com/maiqueb/kubevirt-ipam-claims)
```

Comment:
The PR condition contains some additional important info about it. 

This file is the user guide. Please check if we need to let the user know about it here.

Reply:
done